cmd/cli/cli.go

```diff
@@ -99,12 +99,38 @@ func fileStorageConfig(dsn, options string) (*file.FileStorage, error) {
 }
 
 func mysqlStorageConfig(dsn, options string, logger log.Logger) (*mysql.MySQLStorage, error) {
-	if options != "" {
-		return nil, NoStorageOptions
-	}
+	logger = logger.With("storage", "mysql")
 	opts := []mysql.Option{
 		mysql.WithDSN(dsn),
-		mysql.WithLogger(logger.With("storage", "mysql")),
+		mysql.WithLogger(logger),
+	}
+	if options != "" {
+		for k, v := range splitOptions(options) {
```

Comment:
No good really-really-standard libraries for this?

Reply:
I would love to use the same code that Go uses for `GODEBUG` envvar (that's where the `a=b,c=d` format came from) but it's all private I think. Not sure there's anything in the stdlib that supports us here.